samples/sample_encode/include/pipeline_encode.h

```diff
@@ -320,7 +320,7 @@ class CEncodingPipeline
     MFXFrameAllocator* m_pMFXAllocator;
     mfxAllocatorParams* m_pmfxAllocatorParams;
     MemType m_memType;
-    mfxU16 m_nMemBuffer;
+    mfxU16 m_nPerfOpt;
```

Comment:
please add a comment of the meaning of this variable

Reply:
Done